### PR TITLE
fix(install): don't escape path to scoped packages in subdeps

### LINKED ISSUE #394: Cannot find module for a peerDependency of a devDependency
### Code to reproduce the issue:
```
npm i -g angular-cli
ng new myproject --skip-npm
cd myproject
pnpm i
ng build
```
EDIT:
Faster way to reproduce, thanks to @zkochan :
```
git clone https://github.com/zkochan/pnpm-issue394.git
cd pnpm-issue394
ng build
```

### Expected behavior:
A folder dist is created in the project directory containing the app to deploy.

### Actual behavior:
The command fails with the error: **Cannot find module 'typescript'**

### Additional information:
```
Error: Cannot find module 'typescript'
    at Function.Module._resolveFilename (module.js:455:15)
    at Function.Module._load (module.js:403:25)
    at Module.require (module.js:483:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (C:\Users\aecz\.pnpm-store\@ngtools+webpack@1.0.0\_\src\plugin.js:2:10)
```
Typescript dependency is correctly linked in myproject/node_modules but @ngtools/webpack do not "see" it and declares it as a peerDependency. Maybe because it is itself a dependency of angular-cli. Note that angular-cli is also correctly linked locally as the project declares it as devDependency.

Node v6.5.0 on Windows 7

### PR DESCRIPTION
**Pull request by zkochan**
Close #394

### CODE REVIEW

File: src/install/index.ts [hunk 1]
```diff
@@ -263,11 +263,7 @@ async function symlinkSelf (target: string, pkg: Package, depth: number) {
   await mkdirp(path.join(target, 'node_modules'))
   await relSymlink(
     path.join('..', '_'),
-    path.join(target, 'node_modules', escapeName(pkg.name)))
-}
-
-function escapeName (name: string) {
-  return name && name.replace('/', '%2f')
+    path.join(target, 'node_modules', pkg.name))
 }
```
Comment (iamstarkov) [inline src/install/index.ts:270]: looks more cleaner now

File: src/install/index.ts [hunk 2]
```diff
@@ -263,11 +263,7 @@ async function symlinkSelf (target: string, pkg: Package, depth: number) {
   await mkdirp(path.join(target, 'node_modules'))
   await relSymlink(
     path.join('..', '_'),
```
Comment (aecz) [inline src/install/index.ts]: We have to check if `pkg.name` contains `/` (scoped package) and therefore change `..` to `../..`. Or replace `..` with `target`.

**Comment by timdp** [comment]
I'm currently getting "Module @org/pkg@version not found" on private modules. Is this PR supposed to solve that? I installed pnpm from a Git clone of this branch but the issue seems to persist.

**Comment by zkochan** [comment]
Maybe, but I am not sure. I your private package used by another dependency?

**Comment by timdp** [comment]
No. I'm running `pnpm install` inside the root of a package that depends directly upon the failing package.

**Comment by zkochan** [comment]
Could you please post the content of the `pnpm-debug.log` to gist?

**Comment by timdp** [comment]
I could but I'd have to obfuscate a few package names. It's pretty clear though. It's still replacing slashes with %2F:

```
  251 http 404: "https://registry.npmjs.org/@org-name%2Fpackage-name"
```

**Comment by zkochan** [comment]
It should encode the package name when it requests a package from the registry, so this is different. You can create a separate issue, but try to provide more details, please.

**Comment by timdp** [comment]
Will do!

**Pull request merged**